content/applications/essentials/contacts/merge.rst

```diff
@@ -0,0 +1,77 @@
+==============
+Merge contacts
+==============
+
+Odoo's *Contacts* application allows user's to merge duplicate contacts, without losing any
```

Comment:
```suggestion
Odoo's *Contacts* application provides the ability to merge duplicate contacts, without losing any
```

Reply:
It's been my understanding that we should avoid using the term "user" whenever possible. Hence, this suggestion.

Reply:
I think it is okay in this context; we just need to avoid using "user" in the sense of "the user then clicks..." etc.